### DISCUSSION TITLE
Why 31MB?

### DISCUSSION THREAD

**Discussion by mikeumus** [rxhanson/Rectangle#459] [Q&A] [answered]
Spectacle is only 4MB. Just wondering the size difference? 

Thank you 🙇🏻‍♂️

**Comment by rxhanson** [comment, marked as answer]
This is primarily due to bundling a Swift runtime (I don't remember precisely what it was called) with the app. This is automatically done / mandatory if you support versions of macOS/OSX prior to 10.14. 

Maybe at some point it will make sense to separate out builds for older macs.

Beyond that, there's a few things here or there that might cause a larger file size - more window actions means more images, etc.
